README.md

```diff
@@ -314,12 +314,17 @@ List of deprecated configuration parameters:
 - `output_fields` - A comma separated list of fields to include in output. Supported values are: `key`, `offset`, `timestamp` and `value`. Defaults to `value`.
 
 List of new configuration parameters:
-- `aws.access.key.id` - AWS Access Key ID for accessing S3 bucket. Mandatory.
-- `aws.secret.access.key` - AWS S3 Secret Access Key. Mandatory.
+- `aws.access.key.id` - AWS Access Key ID for accessing S3 bucket.
+- `aws.secret.access.key` - AWS S3 Secret Access Key.
```

Comment:
I think the assume role capability needs some text description apart from just configuration options. Let's add a short section to the README describing authentication options, i.e. the key and role assumption.